apps/frontend/src/pages/analytics/libs/constants/thirty-days.constant.ts

```diff
@@ -0,0 +1,3 @@
+const THIRTY_DAYS = 30;
```

Comment:
there is no point in this name 😅 , it should describe what this is used for
```suggestion
const ANALYTICS_DATE_MAX_RANGE = 30;
```